R/code.R

```diff
@@ -1,14 +1,47 @@
 # The rest of this documentation is in case.R
-#' @param code_name The name/path of the test code. Codes can be named with your
+#' @param name The name/path of the test code. Codes can be named with your
 #'   file system separator and will be organized as a directory structure. Codes
-#'   are located at `./inst/validation/codes/{code_name}`.
+#'   are located at `./inst/validation/test_code/{name}`.
 #'
 #' @export
 #'
 #' @rdname new_item
-vt_use_code <- function(code_name, pkg = ".") {
+#' @importFrom usethis edit_file
+vt_use_test_code <- function(name, username = vt_username(), open = interactive(), pkg = ".") {
 
-  code_path <- create_item(pkg, "codes", code_name)
+  code_name <- vt_set_ext(name, ext = "R")
+
+  # Create file to write in
+  code_name <- create_item(pkg, "test_code", code_name)
+
+  ## if the file didnt exist before, populate with contents
+  if (file.size(code_name) == 0){
+
+    # Create the content to write
+    content <- paste0(c(
+      "",
+      "# Test setup",
```

Comment:
`context()`

Reply:
`context()` is [deprecated in the third edition](https://testthat.r-lib.org/reference/context.html) and it sounds like generally its use is discouraged now. Do we use `context()` for anything in our current code?

Reply:
gtk. our legacy scrape function expects it 😢, luckily we haven't implemented anything in {valtools} yet...